### PR TITLE
DOCU-2415: updates Services and Routes getting started section

### PR DESCRIPTION
**Pull request by rspurgeon**
### Summary
Brings the Services and Routes section of the getting started guide up to date w/ the Get Kong page.

This is PR 2 in a series that will bring the entire Getting Started section together as a tutorial like unit.

### Reason
Improve the getting started experience for docs users.

### Testing
Built docs locally

### CODE REVIEW

**Comment by rspurgeon** [comment]
Document is still WIP

**Comment by rspurgeon** [comment]
I believe this PR will require some rebasing after https://github.com/Kong/docs.konghq.com/pull/4284 and potentially resolving conflicts in TOC and moved urls documents.

File: src/gateway/get-started/services-and-routes.md [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661290122,
+     "updated_at": 1661290122,
+     "tls_verify_depth": null,
+     "id": "c1e98b2b-6e77-476c-82ca-a5f1fb877e07",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
```
Comment (Guaris) [inline src/gateway/get-started/services-and-routes.md:162]: I think this one can probably be replaced with a variation of "If the request was successful you will receive a `200` response and a response body containing the current configuration."

File: src/gateway/get-started/services-and-routes.md [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+     "tags": null,
+     "client_certificate": null,
+     "tls_verify": null,
+     "created_at": 1661290122,
+     "updated_at": 1661292428,
+     "tls_verify_depth": null,
+     "id": "c1e98b2b-6e77-476c-82ca-a5f1fb877e07",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
```
Comment (Guaris) [inline src/gateway/get-started/services-and-routes.md:199]:
```suggestion
   {
     "retries": 6,
   }
```

How do you feel about something like this focusing the reader into the change you want them to see?

File: src/gateway/get-started/services-and-routes.md [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+   ```
+
+1. **Listing services**
+
+   You can list all current services by sending a `GET` request on the base `/services` URL.
```
Comment (Guaris) [inline src/gateway/get-started/services-and-routes.md:204]:
```suggestion
   You can list all current services by sending a `GET` request to the base `/services` URL.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

File: src/gateway/get-started/services-and-routes.md (hunk 2 above)
Comment (rspurgeon) [reply src/gateway/get-started/services-and-routes.md:199]: I can make a change like this, but I suggest we indicate it's a snippet of the full response.  I've done this elsewhere with:

```
...
"retries": 6,
...
```

File: src/gateway/get-started/services-and-routes.md (hunk 1 above)
Comment (rspurgeon) [reply src/gateway/get-started/services-and-routes.md:162]: I do think the point of this step is to show how to view the configuration, so I would prefer to keep this here, but I can shorten what's shown and use a snippet of the important parts instead to reduce the length

File: src/gateway/get-started/services-and-routes.md [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+     "id": "3b2be74e-335b-4f25-9f08-6c41b4720315",
+     "write_timeout": 60000,
+     "ca_certificates": null
+   }
+   ```
+   
+   Fields that are not explicitly provided in the create request are automatically given 
+   a default value based on the current {{site.base_gateway}} configuration.  
+   The [Admin API documentation](/gateway/latest/admin-api/#service-object) has the 
+   full specification for managing service objects.
```
[the remark this answers is not in the document]
Comment (rspurgeon) [reply src/gateway/get-started/services-and-routes.md]: See latest

File: src/gateway/get-started/services-and-routes.md (hunk 1 above)
Comment (rspurgeon) [reply src/gateway/get-started/services-and-routes.md]: Please see latest

File: src/gateway/get-started/services-and-routes.md (hunk 2 above)
Comment (rspurgeon) [reply src/gateway/get-started/services-and-routes.md]: Please see latest

**Comment by rspurgeon** [comment]
rebased on `release/gateway-3.0` and force pushed

File: src/gateway/get-started/services-and-routes.md [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+
+   Like services, when you create a route, {{site.base_gateway}} 
+   assigns it a unique `id` as shown in the response above. The `id` field, or the name provided
+   when creating the route, can be used to identify the route in subsequent requests.
+   The route URL can take either of the following forms:
+   
+   * `/services/{service name or id}/routes/{route name or id}`
+   * `/routes/{route name or id}`
+
+   To view the current state of the `mocking` route, make a `GET` request on the route URL:
```
Comment (Guaris) [inline src/gateway/get-started/services-and-routes.md:248]:
```suggestion
   To view the current state of the `mocking` route, make a `GET` request to the route URL:
```

**Review by Guaris — CHANGES_REQUESTED** [review]
Small nit

**Review by Guaris — APPROVED** [review]

**Pull request merged**